lib/Common/Memory/Recycler.cpp

```diff
@@ -4806,16 +4864,25 @@ Recycler::StartBackgroundMark(bool foregroundResetMark, bool foregroundFindRoots
     bool doBackgroundFindRoots = true;
     if (foregroundResetMark || foregroundFindRoots)
     {
-        RECYCLER_PROFILE_EXEC_BEGIN(this, Js::ResetWriteWatchPhase);
-        bool hasWriteWatch = (recyclerPageAllocator.ResetWriteWatch() && recyclerLargeBlockPageAllocator.ResetWriteWatch());
-        RECYCLER_PROFILE_EXEC_END(this, Js::ResetWriteWatchPhase);
-
-        if (!hasWriteWatch)
+        // REVIEW: SWB, if there's only write barrier page change, we don't scan and mark?
```

Comment:
see comment bellow

---
In reply to: [88137787](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r88137787) [](ancestors = 88137787)